src/main/scala/example/Task1RepartitioningApproach.scala

```diff
@@ -0,0 +1,85 @@
+package example
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.functions.col
+
```

Comment:
can you add Scala doc to all of the methods, objects, classes, etc.?

Reply:
I will.